chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -48,14 +62,15 @@ def forward_cpu(self, inputs):
         if count == 0:
             return numpy.zeros((), dtype=x.dtype),
 
-        y = (numpy.log(p) * (t.flat != self.ignore_label)).sum(keepdims=True) \
+        y = (log_p * (t.flat != self.ignore_label)).sum(keepdims=True) \
             * (-1.0 / count)
```

Comment:
Use `force_type` because `(-1.0 / count)` returns the value which is numpy.float64 type.